From 0a0428c5bd89cb4b3e835a6d0e2f69eb4892d8a0 Mon Sep 17 00:00:00 2001
From: github username <oscar8atech@gmail.com>
Date: Fri, 3 Jan 2020 15:36:24 -0500
Subject: [PATCH] create patch file

---
 client3.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/client3.py b/client3.py
index c4a5911..0ffcc9f 100644
--- a/client3.py
+++ b/client3.py
@@ -50,12 +50,14 @@ def getRatio(price_a, price_b):
 if __name__ == "__main__":
 
 	# Query the price once every N seconds.
-	for _ in iter(range(N)):
+	for _ in range(N):
 		quotes = json.loads(urllib.request.urlopen(QUERY.format(random.random())).read())
 
 		""" ----------- Update to get the ratio --------------- """
+		prices = {}
 		for quote in quotes:
 			stock, bid_price, ask_price, price = getDataPoint(quote)
+			prices[stock] = price
 			print ("Quoted %s at (bid:%s, ask:%s, price:%s)" % (stock, bid_price, ask_price, price))
 
-		print ("Ratio %s" % getRatio(prices['ABC'], prices['DEF'])
+		print ("Ratio %s" % (getRatio(prices['ABC'], prices['DEF'])))
\ No newline at end of file
-- 
2.21.0 (Apple Git-122)

